Calibration/HcalCalibAlgos/plugins/HOCalibAnalyzer.cc

```diff
@@ -2410,12 +2390,6 @@ void HOCalibAnalyzer::endJob() {
                      << parserr[2] << " " << parserr[3] << " " << parserr[4] << " " << parserr[5] << " " << parserr[6]
                      << std::endl;
 
-            double diff = fitres[4] - fitres[1];
```

Comment:
I suspect this `diff`, as well as the `error` below, were intended to be used to fill the histograms below instead of the "unprotected" corresponding quantities, eg at [line 2405](https://github.com/perrotta/cmssw/blob/c798b49bf182ee34fce13a220bc94bd9a0c40902/Calibration/HcalCalibAlgos/plugins/HOCalibAnalyzer.cc#L2405).
As of now they are never used in the code, and can be removed. If my suspect is correct, however, they could be re-inserted, and used appropriately where supposed to be needed